core/storage/classifier/gae_models.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Models for storing the classification models."""
+
+from core.platform import models
+import utils
+
+from google.appengine.ext import ndb
+
+(base_models,) = models.Registry.import_models([models.NAMES.base_model])
+
+
+class ClassifierModel(base_models.BaseModel):
+    """Storage model for classifier used for answer classification.
```

Comment:
Is this a write-once-only model? If so, would be good to document it here.

In any case, err on being verbose in the documentation about how this model is meant to be used, especially if it helps future devs get the context they need.